02/PrintRecord.cpp

```diff
@@ -9,16 +9,16 @@ namespace samples
 {
 	void PrintRecordExample()
 	{
-		string record = "Pope Kim A12345678 18";
+		string firstName = "Pope";
+		string lastName = "Kim";
+		string studentID = "A12345678";
+		string score = "80";
 
-		istringstream inputStream(record);
+		istringstream inputStream;
```

Comment:
hmmm you did not understand m e properly


istringstream inputStream("Pope Kim A12345678 18")

inputStream >> firstName >> lastName >> studentID >> score; 

is what i wanted